packages/slo-stat-bot/src/slo-label.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {GitHubAPI} from 'probot/lib/github';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const config = require('./../data/config.json');
+
+interface SLOStatus {
+  appliesTo: boolean;
+  isCompliant: boolean | null;
+}
+
+/**
+ * Function gets ooslo label name in repo from the config file
+ * @returns the name of ooslo label
+ */
+handle_labeling.getLabelName = async function (): Promise<string> {
+  try {
+    return config.name;
+  } catch (err) {
+    throw 'Unable to get ooslo name from config-label file';
+  }
+};
+
+/**
+ * Function adds ooslo label to the given issue or pr.
+ * Throws an error if label does not exist in repo
+ * @param github unique installation id for each function
+ * @param owner of issue or pr
+ * @param repo of issue or pr
+ * @param issueNumber number of issue pr
+ * @param name of ooslo label in repo
+ * @returns void
+ */
+handle_labeling.addLabel = async function addLabel(
+  github: GitHubAPI,
+  owner: string,
+  repo: string,
+  issueNumber: number,
+  name: string
+) {
+  try {
+    const labels = [name];
+    await github.issues.addLabels({
+      owner,
+      repo,
+      issue_number: issueNumber,
+      labels,
+    });
+  } catch (err) {
+    //Error if ooslo label does not exist in repo
+    throw `Error in adding ooslo label for org ${owner} in repo ${repo} since it does not exist \n ${err}`;
```

Comment:
In cases where you catch an exception, and rethrow, it's important to preserve the context of the original exception.  The original exception will have a full call stack, original error, and all the details that explain what *really* happened here.  I suggest a pattern like this:

```js
} catch (err) {
  err.message = `Error in adding ooslo label for org ${owner} in repo ${repo} since it does not exist \n ${err.message} \n ${err}`;
  throw err;
}
```